osdp/constructs/ui_construct.py

```diff
@@ -78,20 +149,16 @@ def __init__(self, scope: Construct, id: str, *, stack_id: str, api_url: str, **
             )
         )
 
-        # Does it need this?
         self.build_function_url = self.build_function.add_function_url(
             auth_type=_lambda.FunctionUrlAuthType.NONE,
```

Comment:
Can you change this to `auth_type=_lambda.FunctionUrlAuthType.AWS_IAM` (we're triggering NUIT's alarms with wide open function urls).

Reply:
Updated! https://github.com/nulib/osdp-prototype-cdk/pull/6/commits/721014279dd2c6c83a0fe7e3cb432945157904a8